pages/threeView.js

```diff
@@ -131,7 +131,8 @@ window.ThreeView = function() {
             ]);
             envMap.format = THREE.RGBFormat;
             object.traverse(function(node) {
-                if (node.material && 'envMap' in node.material) {
+                // The last part of this guard uses MeshBasicMaterial to look for KHR_materials_unlit.
+                if (node.material && 'envMap' in node.material && node.material.type !== 'MeshBasicMaterial') {
```

Comment:
`node.material.isMeshBasicMaterial` is preferred over checking `.type`. There are also some rare cases where `node.material` could be an array of materials, which I think we'll remove later, but a more robust check would be:

```js
if (!node.isMesh) return;
const materials = Array.isArray(node.material) ? node.material : [node.material];
materials.forEach((material) => {
  if ('envMap' in material && !material.isMeshBasicMaterial) {
    // ...
  }
});
```

Reply:
Thanks!  I pushed an update.